supervisor/errata_utils.py

```diff
@@ -157,6 +157,38 @@ def get_erratum_for_link(link: str, full: bool = False) -> Erratum | FullErratum
     return get_erratum(erratum_id, full=full)
 
 
+class ErratumBuilds(BaseModel):
+    product_list: dict[str, dict[str, set[str]]]
+
+    def __eq__(self, other: object) -> bool:
+        if not isinstance(other, ErratumBuilds):
+            return NotImplemented
+
+        return self.product_list == other.product_list
+
+
+def get_erratum_builds(id: int | str) -> ErratumBuilds:
+    data = ET_api_get(f"erratum/{id}/builds")
```

Comment:
The docs for /builds say:

> See also [/api/v1/erratum/{id}/builds_list](https://errata.engineering.redhat.com/documentation/developer-guide/api-http-api.html#get-apiv1erratumidbuilds_list) which should be preferred over [/api/v1/erratum/{id}/builds](https://errata.engineering.redhat.com/documentation/developer-guide/api-http-api.html#get-apiv1erratumidbuilds).

The difference seems to be unimportant for us, but we should probably use the preferred api.